src/bin/index.ts

```diff
@@ -39,11 +45,22 @@ export async function bin(args: string[]) {
 	const { values } = parseArgs({
 		args,
 		options: {
+			help: {
+				short: "h",
+				type: "boolean",
+			},
 			mode: { type: "string" },
 		},
 		strict: false,
 	});
 
+	const help = values.help;
+
+	if (help) {
```

Comment:
[Style] Nitpick: this is only used once, any particular reason to store in a variable?

```suggestion
	if (values.help) {
```